scripts/schedulers/bash.py

```diff
@@ -0,0 +1,47 @@
+from scripts.schedulers.scheduler import Scheduler
+
+
+class Bash(Scheduler):
+    job_counter = 0
+    task_running = False
+
+    def submit_job(self, script_location: str, target_machine: str = None):
+        """
+        Runs job in the bash shell
+        Will run job until completion
+        Enforces that the script is executable, then executes it
+        :param script_location:
+        :param target_machine:
+        :return: The index of the task
+        """
+        self.logger.debug(
+            f"Ensuring execute set and running script located at: {script_location}"
+        )
+
+        # Could run task as non blocking with internal "schedule manager".
+        # May result in cpu overload if too many tasks are run at once
+        self.task_running = True
+        self._run_command_and_wait(cmd=f"chmod u+x {script_location}")
+        self._run_command_and_wait(cmd=f"{script_location}")
+        self.task_running = False
+        self.job_counter += 1
+        return self.job_counter
+
+    def cancel_job(self, job_id: int, target_machine=None):
+        raise self.raise_scheduler_exception(
```

Comment:
I mean you can. You just terminate the PID. You have that from the submission right?

Reply:
Probably best to add a todo. I don't think we'll be running large workflows on an unmanaged system so don't need the full flexibility.